script/Ciel/Day 10/10_01_CIEL10_1.txt

```diff
@@ -44,7 +44,7 @@ Still, even though my thoughts are in such disarray,
 [sha:ed2df0c7e359fec642534357e47eff7c70e4b876]{
 -- Page 3, Offset 27061.
 -- あの女が、好きなのか。
-Do I... love that woman?
+Do I... love that women?//Thoughts on on changing "That women" to her?
```

Comment:
Editing mistake? Definitely should not be 'that women'

Reply:
oh no, just leaving it until two reviewers check off on the edit.

Reply:
fine with it being changed to her